.github/workflows/push.yml

```diff
@@ -0,0 +1,47 @@
+name: Push or PR
+on: [push, pull_request]
+jobs:
+
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: Set up Go 1.13
+      uses: actions/setup-go@v1
+      with:
+        go-version: 1.13
+      id: go
+
+    - name: Check out code into the Go module directory
+      uses: actions/checkout@v1
+    
+    - name: Set GOPATH
+      # temporary fix
+      # see https://github.com/actions/setup-go/issues/14
```

Comment:
I am looking for a solution for this since GitHub has problems with the environment variables, the solution I wrote found it [here](https://github.com/actions/setup-go/issues/12#issuecomment-524631719), as soon as I find a more appropriate solution or GitHub fix that, I fix it in this part